### PR TITLE
Update ko.json

### PR DESCRIPTION
**Pull request by hoehoetvhss**

### CODE REVIEW

File: i18n/ko.json [hunk 1]
```diff
@@ -938,9 +938,9 @@
     },
     "fact": {
         "fun": {
-            "bigThreeDominance": "재미있는 사실: 방탄소년단, 블랙핑크, 트와이스의 합산 유튜브 조회수는 {{bigThreeViews}}회 또는 {{proportion}}%를 차지 합니다.",
+            "bigThreeDominance": "재미있는 사실: 방탄소년단, 블랙핑크, 트와이스를 합친 유튜브 조회수는 {{bigThreeViews}}회, {{proportion}}%입니다!",
             "birthday": "재미있는 사실: {{name}}의 생일은 이번 달 {{formattedDate}}입니다!",
-            "companyByArtistCount": "재미있는 사실: {{company}}는 총 {{num}}명의 아티스트 (서브유닛 및 솔로 데뷔 포함)가 가장 {{ordinalNum}}명인 엔터테인먼트 회사입니다!",
+            "companyByArtistCount": "재미있는 사실: {{company}}은(는) 총 {{num}}명의 아티스트 (유닛 및 솔로 데뷔 포함)를 보유하고 있는 엔터테인먼트 회사입니다.",
```
Comment (taahamahdi) [inline i18n/ko.json:943]: ordinalNum is missing

File: i18n/ko.json [hunk 2]
```diff
@@ -962,7 +962,7 @@
             "recentGaonWeekly": "재미있는 사실: {{songName}}은(는) 지난 주 가온 주간 차트에서 {{ordinalNum}}로 가장 높은 차트를 기록한 곡입니다!",
             "recentMusicShowWin": "재미있는 사실: {{winDate}}의 {{musicShow}}에서 {{hyperlink}}가 최근 우승했습니다.",
             "viewsByArtistType": "재미있는 사실: 유튜브의 모든 K-POP 뮤직 비디오에 대한 총 조회수는 {{totalViews}}회이며, 그 중 {{groupViews}}(${{groupProportion}}%)가 그룹에서 발생한 반면 {{soloViews}}($ {{soloProportion}}%)는 솔로 아티스트에서 발생했습니다.",
-            "viewsByGender": "재미있는 사실: 재미있는 사실: 유튜브의 모든 K-POP 뮤직 비디오에 대한 총 조회수는 {{totalViews}}회이며, 그 중 {{maleViews}}({{maleProportion}}%)는 남성 그룹, {{femaleViews}} ({{femaleProportion}}%)는 여성 그룹에서, 나머지 {{coedViews}}({{coedProportion}}%)는 남녀 혼성 그룹에서 발생했습니다."
+            "viewsByGender": "재미있는 사실: 유튜브의 모든 K-POP 뮤직 비디오에 대한 총 조회수는 {{totalViews}}회이며, 그 중 {{maleViews}}({{maleProportion}}%)는 남성 그룹, {{femaleViews}} ({{femaleProportion}}%)는 여성 그룹에서, 나머지 {{coedViews}}({{coedProportion}}%)는 남녀 혼성 그룹에서 발생했습니다."
```
Comment (taahamahdi) [inline i18n/ko.json:965]: There should probably be a space between {{maleViews}} and ({{maleProportion}}), as well as {{coedViews}} and ({{coedProportion}})

**Review by taahamahdi — CHANGES_REQUESTED** [review]

File: i18n/ko.json (hunk 2 above)
Comment (hoehoetvhss) [reply i18n/ko.json]: We have made corrections to this point.

File: i18n/ko.json (hunk 1 above)
Comment (hoehoetvhss) [reply i18n/ko.json:943]: Subtracting `{{ordinalNum}}` is intentional, as it seems more appropriate for Korean sentences.

Comment (hoehoetvhss) [reply i18n/ko.json]: add, We have made corrections to this point.

**Comment by hoehoetvhss** [comment]
We decided that all parts of the fun facts should be re-corrected, and the fun fact correction will proceed in `version 1.2`

**※ Among the contents that I have worked on, the contents to be modified are to focus on the modification.**

**Review by taahamahdi — APPROVED** [review]

**Pull request merged**